vspreview/core/vsenv.py

```diff
@@ -41,7 +42,45 @@ def new_environment(self) -> ManagedEnvironment:
 
 
 def _monkey_runpy_func(*args: Any, **kwargs: Any) -> Any:
-    glob_dict = orig_runpy_run_code(*args, **kwargs)
+    # TODO: Update the guide to better separate plugins and Python packages, then update this URL
+    base_url = 'https://jaded-encoding-thaumaturgy.github.io/JET-guide/master'
+    guide_url = f'{base_url}/basics/installation/#installing-vapoursynth-plugins'
+
+    try:
+        glob_dict = orig_runpy_run_code(*args, **kwargs)
+    except (AttributeError, ModuleNotFoundError) as e:
+        is_attr = isinstance(e, AttributeError)
+
+        if is_attr:
+            dep = str(e).partition('named ')[2].partition('.')[0].strip()
+        else:
+            dep = str(e).partition('\'')[2].rstrip('\'').strip()
+
+        raise DependencyNotFoundError(
+            'vspreview', dep,
+            f'Could not find the {("plugin" if is_attr else "Python package")}, \'{dep}\'! '
```

Comment:
If this PR does proceed, consider phrasing this even more specifically as “VapourSynth plugin”, to reduce confusion with vspreview plugins.